todo/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux'
+import rootReducer from './reducers'
+
+ReactDOM.render(
+    <Provider store={store}>
```

Comment:
Store isn't actually being created here. Check this out: https://redux.js.org/api-reference/createstore for more about how to createStore.